x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_validators.test.tsx

```diff
@@ -72,7 +72,7 @@ describe('Output form validation', () => {
       const res = validateLogstashHosts(['https://test.fr:5044']);
 
       expect(res).toEqual([
-        { index: 0, message: 'Invalid logstash host should not start with http(s)' },
+        { index: 0, message: 'Host URL must begin with a domain name or IP address' },
```

Comment:
```suggestion
        { index: 0, message: 'Host URL must begin with a domain name or IP address.' },
```

Reply:
@gchaps I didn't change these messages to include a period because it looks like other validation messages in Fleet do not. I was going to wait to make these changes, but if you think it's right to follow the guidelines (and be inconsistent with other parts of the Fleet UI), I will apply your changes. Let me know.

Reply:
You're right.  Best to keep the Fleet UI consistent with itself, and then make all these changes at once.